apps/prairielearn/src/tests/fileEditor.test.ts

```diff
@@ -299,10 +297,6 @@ describe('test file editor', function () {
       });
     });
 
-    describe('disallow edits outside course directory', function () {
-      badGet(badPathUrl, 500, false);
-    });
```

Comment:
Unlike `request`, `fetch(...)` seems to pre-normalize URLs before sending them, so the `/file_edit/../` part of the URL is normalized away and we end up with a 404 (since the normalized URL doesn't exist) instead of a 500 as expected.

What do folks think, should we jump through hoops to retain this test?

Reply:
It's a bit unfortunate to lose a security test like this, but I don't want to go to some extreme effort to retain it. If there is some easy way to do a fetch (or equivalent) with `/../` in the path then I'd retain it, but otherwise 🤷

Reply:
I was able to add the test back in 63de96e by using `node:http` and `node:url` directly. How does that look?

Reply:
Looks great, thanks for the effort to preserve this test!